packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -179,6 +183,42 @@ async function makePsmDriver(t, customTerms) {
     }),
   );
 
+  /**
+   * @param {Amount<'nat'>} giveAnchor
+   * @param {Amount<'nat'>} [wantStable]
+   */
+  const swapAnchorForStableSeat = async (giveAnchor, wantStable) => {
+    const seat = E(zoe).offer(
+      E(publicFacet).makeWantStableInvitation(),
+      harden({
+        give: { In: giveAnchor },
+        ...(wantStable ? { want: { Out: wantStable } } : {}),
```

Comment:
I like this.  It's a definite improvement on my approach of making the proposal, and then adding the want later.